.travis.yml

```diff
@@ -4,5 +4,9 @@ jdk:
 before_script:
```

Comment:
Please remove the before_script section, this project is not related to databases.